site-src/guides/examples.md

```diff
@@ -0,0 +1,156 @@
+## Example AdminNetworkPolicy API Object Definitions 
+
+These examples will start with the object yaml defintions used to implement the 
+[core use cases](../index.md#adminnetworkpolicy-api-user-stories). Please feel 
+free to contribute more examples that may seem relevant to other users :-). 
+
+### Sample Spec for Story 1: Deny traffic at a cluster level
```

Comment:
for all of these I think we should add brief descriptions of the ANP's at work, to go with the image

Reply:
also we should add something to show the priority at work to show a BANP getting overridden or something similar to that

Reply:
another option would be to have a better link between the cases and these examples

Reply:
+1 lets tackle this in a follow up